core/domain/exp_services.py

```diff
@@ -104,7 +107,17 @@ def _migrate_states_schema(versioned_exploration_states):
 
 # Repository GET methods.
 def _get_exploration_memcache_key(exploration_id, version=None):
-    """Returns a memcache key for an exploration."""
+    """Returns a memcache key for an exploration.'
+    
+    Args:
+        exploration_id: The exploration id in question.
```

Comment:
Let's follow a convention here: if the arg type is simple to state, let's state it. So, in this case, it would be:

Args:
    exploration_id: str. The exploration id whose memcache key should be retrieved.
    version: ??? or None. If specified, the returned memcache key corresponds to this particular version of the exploration. Otherwise, a memcache key that represents the latest version of the exploration is returned.

Note that I don't actually know what ??? is -- you would need to find out if it's str or int. This is sort of thing the docstring is really useful for, btw!